Tests/SentryTests/SentryOptionsTest.m

```diff
@@ -763,11 +765,73 @@ - (void)testEnableSwizzling
     [self testBooleanField:@"enableSwizzling"];
 }
 
+- (void)testEnableTracing
```

Comment:
Excellent tests. They helped me to understand the impact of the implementation 🥳